ts/screens/wallet/payment/PickPspScreen.tsx

```diff
@@ -102,66 +104,98 @@ class PickPspScreen extends React.Component<Props> {
+              source={{ uri: item.logoPSP }}
+              onError={this.onErrorImageLoading}
+            />
+          ) : (
+            <Body>{item.businessName}</Body>
+          )}
+          <View style={styles.feeContainer}>
+            <Label>{formatNumberCentsToAmount(item.fixedCost.amount)}</Label>
+            <IconFont
+              name="io-right"
+              size={ICON_SIZE}
+              color={customVariables.contentPrimaryBackground}
+            />
+          </View>
         </View>
-        <Text style={styles.feeText}>
-          {`${I18n.t("wallet.pickPsp.maxFee")} `}
-          <Text bold={true} style={styles.feeText}>
-            {formatNumberCentsToAmount(item.fixedCost.amount)}
-          </Text>
-        </Text>
       </TouchableDefaultOpacity>
     );
   };
 
   public render(): React.ReactNode {
     const availablePsps = orderPspByAmount(this.props.allPsps);
 
+    const backButtonProps = {
+      block: true,
+      primary: true,
+      bordered: true,
+      onPress: this.props.navigateBack,
+      title: I18n.t("global.buttons.back")
+    };
+
     return (
       <BaseScreenComponent
         goBack={true}
-        headerTitle={I18n.t("wallet.pickPsp.title")}
+        headerTitle={I18n.t("wallet.pickPsp.headerTitle")}
         contextualHelpMarkdown={contextualHelpMarkdown}
         faqCategories={["payment"]}
       >
-        <Content noPadded={true}>
-          <View spacer={true} />
+        <Content noPadded>
+          <View spacer />
           <View style={styles.padded}>
-            <Text>
-              <Text bold={true}>{`${I18n.t("wallet.pickPsp.infoBold")} `}</Text>
-              {`${I18n.t("wallet.pickPsp.info2")} `}
-            </Text>
-            <Text link={true} onPress={this.showHelp}>
-              {I18n.t("wallet.pickPsp.link")}
-            </Text>
+            <H3>{I18n.t("wallet.pickPsp.title")}</H3>
+            <View spacer small />
+            <Label weight="Regular" color="bluegreyDark">
+              {I18n.t("wallet.pickPsp.info")}
+            </Label>
+            <Label weight="Regular" color="bluegreyDark">
+              {I18n.t("wallet.pickPsp.info2")}
+              <Label weight="Bold" color="bluegreyDark">{` ${I18n.t(
+                "wallet.pickPsp.info2Bold"
+              )}`}</Label>
+            </Label>
           </View>
-          <View spacer={true} />
+          <View spacer />
           <FlatList
             ItemSeparatorComponent={() => <ItemSeparatorComponent />}
             removeClippedSubviews={false}
             data={availablePsps}
             keyExtractor={item => item.id.toString()}
             renderItem={this.getListItem}
-            ListFooterComponent={<EdgeBorderComponent />}
+            ListHeaderComponent={this.headerItem}
+            ListFooterComponent={() => <ItemSeparatorComponent />}
           />
         </Content>
+        <FooterWithButtons type="SingleButton" leftButton={backButtonProps} />
```

Comment:
There is already a function that returns the props you set in `backButtonProps`. Can you use it?
```suggestion
        <FooterWithButtons
            type="SingleButton"
            leftButton={cancelButtonProps(
              this.props.navigateBack,
              I18n.t("global.buttons.back")
            )}
          />
```